
package com.teamsystem.hub.b2b.soap.read.v7;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.1.10
 * 2018-10-28T14:46:11.642+01:00
 * Generated source version: 3.1.10
 */

@WebFault(name = "authenticationException", targetNamespace = "http://schema.read.b2b.hub.teamsystem.com/v7")
public class AuthenticationExceptionMsg extends Exception {
    
    private com.teamsystem.hub.b2b.soap.read.v7.AuthenticationException authenticationException;

    public AuthenticationExceptionMsg() {
        super();
    }
    
    public AuthenticationExceptionMsg(String message) {
        super(message);
    }
    
    public AuthenticationExceptionMsg(String message, Throwable cause) {
        super(message, cause);
    }

    public AuthenticationExceptionMsg(String message, com.teamsystem.hub.b2b.soap.read.v7.AuthenticationException authenticationException) {
        super(message);
        this.authenticationException = authenticationException;
    }

    public AuthenticationExceptionMsg(String message, com.teamsystem.hub.b2b.soap.read.v7.AuthenticationException authenticationException, Throwable cause) {
        super(message, cause);
        this.authenticationException = authenticationException;
    }

    public com.teamsystem.hub.b2b.soap.read.v7.AuthenticationException getFaultInfo() {
        return this.authenticationException;
    }
}
